scripts/simpledvt/02-clone-nor.js

```diff
@@ -0,0 +1,474 @@
+const DEPLOYER = process.env.DEPLOYER || ''
+
+const SIMULATE = !!process.env.SIMULATE
+const VOTE_ID = process.env.VOTE_ID || ''
+
+const REQUIRED_NET_STATE = [
+  'ensAddress',
+  'lidoApmAddress',
+  'lidoApmEnsName',
+  'lidoLocator',
+  `app:${APP_NAMES.ARAGON_VOTING}`,
+  `app:${APP_NAMES.ARAGON_TOKEN_MANAGER}`,
+]
+
+async function deployNORClone({ web3, artifacts, trgAppName = APP_TRG, ipfsCid = APP_IPFS_CID }) {
+  const netId = await web3.eth.net.getId()
+
+  const srcAppName = APP_NAMES.NODE_OPERATORS_REGISTRY
+
+  log.splitter()
+  log(`Network ID: ${chalk.yellow(netId)}`)
+
+  const deployer = await getDeployer(web3, DEPLOYER)
+  const state = readNetworkState(network.name, netId)
+  assertRequiredNetworkState(state, REQUIRED_NET_STATE.concat([`app:${srcAppName}`, `app:${trgAppName}`]))
+
+  const kernelAddress = state.daoAddress || readStateAppAddress(state, `aragon-kernel`)
+  if (!kernelAddress) {
+    throw new Error(`No Aragon kernel (DAO address) found!`)
+  }
+
+  log.splitter()
+
+  log(`Using ENS:`, yl(state.ensAddress))
+  const ens = await artifacts.require('ENS').at(state.ensAddress)
+  const lidoLocatorAddress = readStateAppAddress(state, `lidoLocator`)
+  log(`Lido Locator:`, yl(lidoLocatorAddress))
+  log.splitter()
+
+  const srcAppFullName = `${srcAppName}.${state.lidoApmEnsName}`
+  const srcAppId = namehash(srcAppFullName)
+  const { semanticVersion, contractAddress } = await resolveLatestVersion(srcAppId, ens, artifacts)
+  const srcVersion = semanticVersion.map((n) => n.toNumber())
+
+  log(`Source App:`, yl(srcAppName))
+  log(`Source App ENS:`, yl(srcAppFullName))
+  log(`Source App ID:`, yl(srcAppId))
+  log(`Source Contract implementation:`, yl(contractAddress))
+  log(`Source App version:`, yl(srcVersion.join('.')))
+  log.splitter()
+
+  const trgAppFullName = `${trgAppName}.${state.lidoApmEnsName}`
+  const trgAppId = namehash(trgAppFullName)
+  const trgProxyAddress = readStateAppAddress(state, `app:${trgAppName}`)
+  const trgAppArtifact = APP_ARTIFACTS[srcAppName] // get source app artifact
+  const trgApp = await artifacts.require(trgAppArtifact).at(trgProxyAddress)
+
+  // set new version to 1.0.0
+  const trgVersion = [1, 0, 0]
+  const contentURI = '0x' + Buffer.from(`ipfs:${ipfsCid}`, 'utf8').toString('hex')
+
+  log(`Target App:`, yl(trgAppName))
+  log(`Target App ENS:`, yl(trgAppFullName))
+  log(`Target App ID:`, yl(trgAppId))
+  log(`Target App proxy`, yl(trgProxyAddress))
+  log(`Target Contract implementation:`, yl(contractAddress))
+  log(`Target Content IPFS CID:`, yl(ipfsCid))
+  log(`Target Content URI:`, yl(contentURI))
+  log(`Target App version:`, yl(trgVersion.join('.')))
+
+  log.splitter()
+  const {
+    moduleName,
+    moduleType = 'curated',
+    targetShare = 1000,
+    moduleFee = 500,
+    treasuryFee = 500,
+    penaltyDelay,
+    easyTrackAddress,
+    easyTrackEVMScriptExecutor,
+    easyTrackFactories = {},
```

Comment:
it might be better to have no default values here and crash in case no values are specified